src/view/output-view.js

```diff
@@ -32,24 +32,27 @@ export function processJVMOutput(output, theme) {
     .replace(`${ANGLE_BRACKETS_LEFT_HTML}/errStream${ANGLE_BRACKETS_RIGHT_HTML}`, "</span>");
 }
 
-export function processJUnitResults(data) {
+export function processJUnitResults(data, onTestPassed) {
   let result = "";
   let totalTime = 0;
+  let passed = true;
   if (isEmptyObject(data)) return NO_TEST_FOUND;
   for (let testClass in data) {
     let listOfResults = arrayFrom(data[testClass]);
     result = result + listOfResults.reduce((previousTest, currentTest) => {
       totalTime = totalTime + (currentTest.executionTime / 1000);
+      if (currentTest.status === TEST_STATUS.ERROR.value || currentTest.status === TEST_STATUS.FAIL.value) passed = false;
       switch (currentTest.status) {
-        case "FAIL":
-          return previousTest + `<span class="console-icon fail"></span><div class="test-fail">${TEST_STATUS.FAIL}: ${currentTest.methodName}: ${convertToHtmlTag(currentTest.comparisonFailure.message)}</div>`;
-        case "ERROR":
-          return previousTest + `<span class="console-icon fail"></span><div class="test-fail">${TEST_STATUS.ERROR}: ${currentTest.methodName}: ${convertToHtmlTag(currentTest.exception.message)}</div>`;
-        case "OK":
-          return previousTest + `<span class="console-icon ok"></span><div class="test-output">${TEST_STATUS.PASSED}: ${currentTest.methodName}</div>`;
+        case TEST_STATUS.FAIL.value:
```

Comment:
why don't you use just enum without value prefix

Reply:
because it's not a ts